### PR TITLE
MHV-45005

### PR DESCRIPTION
**Pull request by gerard-sullivan**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary
 implemented the getting image uri logic to frontend because it is not needed to change backend anymore

## Related issue(s)

https://jira.devops.va.gov/browse/MHV-45005

## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |      Screen Shot 2023-09-13 at 12 41 01 PM  |      Screen Shot 2023-09-13 at 12 37 52 PM |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by vmenshutin-bylight — APPROVED** [review]

File: src/applications/mhv/medications/util/helpers.js
Comment (acrollet) [inline src/applications/mhv/medications/util/helpers.js:56]: I'd like to see unit tests for this function to confirm that we get the expected behavior from it.

File: src/applications/mhv/medications/util/helpers.js
Comment (acrollet) [inline src/applications/mhv/medications/util/helpers.js:63]: would it be simpler/more readable to do something like this? (untested pseudocode)

`if (!folderName.match(/^[1-9]$/))`

**Review by acrollet — CHANGES_REQUESTED** [review]
Overall looks good, just a couple of questions/comments 🙂

File: src/applications/mhv/medications/util/helpers.js
Comment (gerard-sullivan) [reply src/applications/mhv/medications/util/helpers.js:63]: it is from original java code of krish.
Screen Shot 2023-09-11 at 9 47 51 AM

File: src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx
Comment (gerard-sullivan) [reply src/applications/mhv/medications/components/PrescriptionDetails/VaPrescription.jsx]: I am waiting for the response from UCD

File: src/applications/mhv/medications/util/helpers.js
Comment (gerard-sullivan) [reply src/applications/mhv/medications/util/helpers.js:56]: I will add unit test in another ticket because it should be merged as soon as possible.
Vitalli wants to avoid conflict with my update.

**Review by acrollet — APPROVED** [review]
leaving approval to unblock for merging dependencies, see: https://dsva.slack.com/archives/C059Q28DV99/p1694643370924619?thread_ts=1694634206.678009&cid=C059Q28DV99

**Pull request merged**
